README.md

```diff
@@ -38,6 +38,12 @@ conda install -c conda-forge r-grf
 
 Note that to install from source, a compiler that implements C++11 is required (clang 3.3 or higher, or g++ 4.8 or higher). If installing on Windows, the RTools toolchain is also required.
 
+The current development version can be installed from source using devtools.
```

Comment:
All published releases can already be installed from the CRAN archive, but it could still be useful to retain the tarballs for easy internal record keeping, and avoiding slip-ups in the release process: the thing that is submitted to CRAN will _always_ be the committed tarball?

Reply:
I don't see a big benefit, but I don't feel strongly about this point (and you are doing most of the releases anyways :))

Reply:
Yes, it is a bit redundant to have them, I just like the very rigid workflow when making a new release (open a new PR, commit things step by step, and have a record of the tarball) :P

Reply:
I will leave this to y'all. Feel free to merge or not.

Reply:
My vote is to remove the "Any published release can also be installed from source..." paragraph from the docs, then merge.